bitnami/elasticsearch/templates/metrics-deploy.yaml

```diff
@@ -43,10 +43,10 @@ spec:
           args:
             {{- if gt (int .Values.coordinating.replicas) 0 }}
             # Prefer coordinating only nodes to do the initial metrics query
-            - --es.uri=http://{{ template "elasticsearch.coordinating.fullname" . }}:{{ .Values.coordinating.service.port }}
+            - --es.uri=http://elastic:{{ .Values.security.elasticPassword }}@{{ template "elasticsearch.coordinating.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.coordinating.service.port }}
```

Comment:
Isko bro, dynamically create karte hai naa.. if auth is enabled then yeh use ho otherwise purana waala ho